packages/phone_log/android/src/main/java/com/jiajiabingcheng/phonelog/CallRecord.java

```diff
@@ -9,6 +9,7 @@
   String formattedNumber;
   String number;
   String callType;
+  String dialedNumber;
```

Comment:
Can you document the difference between the 3 numbers so others don't run into the same issue that we did.

Reply:
I see the comments when we fetch the data but it would be better to have it in this file also.

Reply:
Is there a benefit to distinguishing between these numbers?

If you just want to get the number that was dialed, why not implement a fallback mechanism in the plugin itself rather than forcing all apps to understand what each field does?

Reply:
That's a good point. 

However, one advantage of having the different fields is that they are formatted differently. E.g. the formattedNumber holds a value such as (456) 123 456 (based on user's country settings) and the dialedNumber holds values like 456123456. Also, depending on how the call was placed (directly dialing the number vs clicking on the contact etc.), the number can be present -in different formats- in one of these fields. So based on which number the user is looking for, I think it would be a good idea to expose these different kinds.

Further, this is only from the point of view of making a call; that being said, I think I should rename the 'dialedNumber' variable to something else like unformattedNumber or baseNumber (something on those lines), because it would hold dialed as well as received numbers.

Thoughts?